tex/paper.tex

```diff
@@ -411,41 +373,195 @@ \section*{Description of the Axelrod Python package}\label{sec:description-of-ax
+
+\section*{Additional system requirements}
+
+There are no specific additional system requirements.
+
+\section*{Support}
+
+Support is readily available in multiple forms:
+
+\begin{itemize}[noitemsep,topsep=0pt]
+    \item An online chat channel:
+        \url{https://gitter.im/Axelrod-Python/Axelrod}.
+    \item An  email group:
+        \url{https://groups.google.com/forum/#!topic/axelrod-python}.
+\end{itemize}
+
+\section*{Dependencies}
+
+The following Python libraries are required dependencies:
+
+\begin{multicols}{2}
+    \begin{itemize}[noitemsep,topsep=0pt]
+        \item Numpy 1.9.2
+        \item Matplotlib 1.4.2
+        \item Testfixtures 4.9.1
+        \item Hypothesis 3.0 (only a requirement for development)
+    \end{itemize}
+\end{multicols}
+
+\section*{List of contributors}
+
+The names of all the contributors are not known: as these were mainly done
+through Github and some have not provided their name or responded to a
+request for further details. Here is an incomplete list:
+
+\begin{multicols}{2}
+    \begin{itemize}[noitemsep,topsep=0pt]
+        \item {Owen Campbell}
+        \item {Marc Harper}
+        \item {Vincent Knight}
+        \item {Karol M. Langner}
+        \item {James Campbell}
+        \item {Thomas Campbell}
+        \item {Alex Carney}
+        \item {Martin Chorley}
+        \item {Cameron Davidson-Pilon}
+        \item {Kristian Glass}
+        \item {Tom{\'a}{\v s} Ehrlich}
+        \item {Martin Jones}
+        \item {Georgios Koutsovoulos}
+        \item {Marissa Tibble}
+        \item {Jochen M{\"u}ller}
+        \item {Geraint Palmer}
+        \item {Paul Slavin}
+        \item {Timothy Standen}
+        \item {Luis Visintini}
+        \item {Karl Molden}
+        \item {Jason Young}
+        \item {Andy Boot}
+        \item {Anna Barriscale}
+    \end{itemize}
+\end{multicols}
+
+\section*{Software location:}
+
+{\bf Archive}
+
+\begin{description}[noitemsep,topsep=0pt]
+    \item[Name:] Zenodo
+    \item[Persistent identifier:]
+\href{https://zenodo.org/record/53982}{10.5281/zenodo.53982}
+    \item[Licence:] MIT
+    \item[Publisher:]  Vincent Knight
+    \item[Version published:] Axelrod: 1.1.1
+    \item[Date published:] 2016-06-01
+\end{description}
+
+{\bf Code repository}
+
+\begin{description}[noitemsep,topsep=0pt]
+    \item[Name:] Github
+    \item[Identifier:] \url{https://github.com/Axelrod-Python/Axelrod}
+    \item[Licence:] MIT
+    \item[Date published:] 2015-02-16
+\end{description}
+
+\section*{Reuse potential}\label{sec:reuse}
+
+The Axelrod library has been designed with sustainable software practice in
+mind. There is an extensive documentation suite:
+\url{axelrod.readthedocs.org/en/latest/}. Furthermore, there is a growing set
+of example Jupyter notebooks available here:
+\url{https://github.com/Axelrod-Python/Axelrod-notebooks}.
+
+The ability to have readily available a large number of strategies makes this
```

Comment:
This sentence is awkward... maybe something "The availability of a large number of strategies...